docs/guides/data-movement/readme.md

```diff
@@ -0,0 +1,93 @@
+---
+authors: Blake Devcich <blake.devcich@hpe.com>
+categories: provisioning
+---
+
+# Data Movement Configuration
+
+Data Movement can be configured in multiple ways:
+
+1. Server side
+2. Per Copy Offload API Request arguments
+
+The first method is a "global" configuration - it affects all data movement operations. The second
+is done per the Copy Offload API, which allows for some configuration on a per-case basis - but is
+limited in scope. Both methods are meant to work in tandem.
+
+
+More on each in the sections below.
+
+## Server Side ConfigMap
+
+The server side configuration is done via the `nnf-dm-config` config map:
```

Comment:
Does the nnf-dm installation provide the initial version of this configmap?  It's not clear.